bin/rbenv-each

```diff
@@ -46,7 +71,13 @@ failed_rubies=""
 
 trap "exit 1" INT
 
-for ruby in $(rbenv versions --bare); do
+if [ -n "$aliases" ]; then
+    opts=
+else
+    opts="--skip-aliases"
```

Comment:
Hm, now that I look at `rbenv versions` argument handling in v0.4.0, it looks like it only recognizes `--bare` as 1st argument and ignores everything else. So I'm pretty sure that calling `rbenv-versions --bare --skip-aliases` will be safe for all rbenv versions, but of course only rbenv 1.0 will respect `--skip-aliases`.